earth2studio/models/px/__init__.py

```diff
@@ -22,3 +22,4 @@
 from earth2studio.models.px.pangu import Pangu3, Pangu6, Pangu24
 from earth2studio.models.px.persistence import Persistence
 from earth2studio.models.px.sfno import SFNO
+from earth2studio.models.px.stormcast import StormCast
```

Comment:
it might be prescient of us to mark this as `StormCast-V1`

Reply:
I'm sort of inclined to leave it as `StormCast` for consistency with the paper, and later versions can have a V2. The first GPT was only called GPT-1 after GPT-2 came out :)

Reply:
StormCast is fine if it maps to the paper I imagine, we did this with FourCastNet :+1:

Reply:
Disagree but will approve.